config/theme-tabler.php

```diff
@@ -101,6 +101,11 @@
      * the classes that make up the look of your panel.
      */
     'classes' => [
+        /**
+         * Use this to pass classes to the table wrapper component displayed in List Operation
```

Comment:
table wrapper component , displayed in the list operation ... called body. 

You know this smells hu 👃 # ?

Reply:
Oups, sorry. Wrong comment here - good catch. Fixed in https://github.com/Laravel-Backpack/theme-tabler/commit/b467b94d7563c04ab196c5e855a57ab4a865f25d